packages/driver/test/cypress/integration/commands/actions/check_spec.js

```diff
@@ -0,0 +1,1233 @@
+const $ = Cypress.$.bind(Cypress)
+const {
+  _,
+} = Cypress
+const {
+  Promise,
+} = Cypress
+
+describe('src/cy/commands/actions/check', () => {
+  before(() => {
+    cy
+    .visit('/fixtures/dom.html')
+    .then((win) => {
+      this.body = win.document.body.outerHTML
```

Comment:
`this` with arrow functions, is this maintaining the correct `this`?

Reply:
I confirm that `this.body` is visible from the test. I've tested with inserting a `cy.log(this.body)` inside a test and the html is in the log.

Are you referring to this feature of mocha.js?
https://mochajs.org/#arrow-functions